k2/csrc/fsa_equivalent.h

```diff
@@ -32,6 +32,10 @@ bool IsRandEquivalent(const Fsa &a, const Fsa &b, std::size_t npath = 100);
   @param [in]  a_weights  Arc weights of `a`
   @param [in]  b          The other FSA to be checked the equivalence
   @param [in]  b_weights  Arc weights of `b`
+  @param [in]  beam       beam > 0 that affects pruning in `a`; the algorithm
+                          will keep paths that are within `beam` of the best
+                          path in `a` to check the equivalence. Just keep
+                          `kFloatInfinity` if you don't want pruning.
```

Comment:
so I assume that paths that are not within the beam don't have to be equivalent?  because thats not what the code currently does.

Reply:
Yes, as we only keep those paths within `beam`, like we do `Prune(a, beam)` first in the algorithm.